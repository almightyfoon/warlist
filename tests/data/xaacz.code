c8101b_-0p4ajGjFjF4H4Hg4g45g5gfImVmV5e5ei7___-0jmP4ImJhmmI54mF5qmj5wmj5wmH
