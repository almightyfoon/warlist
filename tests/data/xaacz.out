https://conflictchamber.com/#c8101b_-0p4ajGjFjF4H4Hg4g45g5gfImVmV5e5ei7___-0jmP4ImJhmmI54mF5qmj5wmj5wmH

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Krueger 2] Krueger the Stormlord [+27]
 - Wold Guardian [16]
 - Woldwarden [14]
 - Woldwarden [14]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [3]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Bog Trog Mist Speaker [4]
Gallows Grove [2]
Gallows Grove [2]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Celestial Fulcrum [19]

Circle Army - 75 / 75 points
[Theme] The Devourer's Host

[Iona 1] Iona the Unseen [+28]
 - Gnarlhorn Satyr [12]
 - Warpwolf Stalker [17]
Gremlin Swarm [3]
Lord of the Feast [0(6)]
Tharn Ravager Shaman [4]
Tharn Bloodtrackers (max) [15]
 - Nuala the Huntress [4]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [0(5)]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [0(5)]
Tharn Wolf Riders (max) [18]
